### DISCUSSION TITLE
"Recommended for you" and related suggestions should have more indicative colors

### DISCUSSION THREAD

**Discussion by madebylydia** [community/community#13159] [Other Feature Feedback, Questions, & Ideas] [answered]
Hey there GitHub! First time I'm providing feedback on a new beta functionality, let's say it's a pretty cool step!

So far, I've been liking the new "For you" feed, and I do not mind it as long as you guys keep our good old "Following" tab. (Now thinking about it, I don't think it was called as such before?)

One important part of this new addition is the icons, and what they represent, I like the one with avatar + minimized icon, but I do have a bit of an hassle with the others icons, such as "Recommended for you", "Popular among people you follow" or "Recommended based on people you follow".
The actual colors for these being this:
![image](https://user-images.githubusercontent.com/61093863/159349002-0a951d27-b4e7-432e-b9b2-1451d17951b2.png)
I don't think gray is a great choice of color here, I think they deserve more varieties, in top of that, it is hard to see the icon inside the circle correctly. The icon itself, however, is just fine!

The following is what I think would be a great choice of colors for the icons:

### Popular among people you follow
![image](https://user-images.githubusercontent.com/61093863/159349121-e56d50af-1e7b-4f24-a68c-3c6e54d0ad13.png)
Color used: `#ff6a69`

### Recommended based on people you follow
![image](https://user-images.githubusercontent.com/61093863/159349250-0a58bb65-e1f6-43f9-9713-f0ed8f98aa4e.png)
Color used: `#539bf5`

### Recommended for you
*Actually, another color should be choose, I had a hard time finding a correct and even purple doesn't look correct to me, but it's the best imo.*
![image](https://user-images.githubusercontent.com/61093863/159349811-c8923a1b-82e4-4a2a-a8cd-37f778e982d4.png)
Color used: `#8e52c9`

:+1:

**Comment by jfullstackdev** [comment, marked as answer]
other than that, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

so we can minimize it so we can focus more in each category

**Reply by madebylydia** [reply]
I do not understand how's that related to my post... It's just about colors, not... filtering?

**Reply by jfullstackdev** [reply]
indicative colors can easily catch attention,

so as the filter for each category, 

and I think about the indicative colors, they are now implementing it

![image](https://user-images.githubusercontent.com/47092464/182908796-fa685733-2e0e-4c06-9836-b67b8ec4323c.png)

different from fork, starred and followed with indicative icons too

**Reply by madebylydia** [reply]
Ah, I see. 👍

I didn't notice the changes they've done on "For You" recently, glad colors had some changes here and here. Thanks for letting me know! 😄
